docs/assets/templates/notebooks/template-big.dockstore.yml

```diff
@@ -0,0 +1,74 @@
+# The first line refers to the version 1.2 of the .dockstore.yml schema
+version: 1.2
+
+# An array of notebooks. Each element corresponds to a notebook on Dockstore.
+notebooks:
+  
+  # The optional notebook name for a notebook, which may only consist of alphanumerics
+  # and internal underscores and hyphens, but no spaces or other characters. Names may not exceed 256 characters.
+  # If using a .dockstore.yml with multiple notebooks, this field is required 
+  # to uniquely identify notebooks in the repository.
+  #
+  # It should be noted that having the name come first is an arbitrary decision.
+  # You could use subclass instead, for instance. Provided arrays are not broken
+  # up, the order of fields within a .dockstore.yml is not important.
+  - name: <String>
+    
+    # The format used for the notebook. Currently, only JUPYTER is supported.
+    # This cannot be changed once the notebook is registered.
+    format: <JUPYTER>
+
+    # The programming language of the code contained in the notebook.
+    language: <PYTHON | R | JAVASCRIPT | SCALA | JULIA | OTHER>
+
+    # The absolute path to the notebook file in the Git repository.
+    # - For Jupyter notebooks, the notebook is a .ipynb file.
+    path: <String>
+
+    # The kernel image reference for the notebook.
+    kernel: <String>
+    
+    # Notebook-wide setting that will affect ALL branches/tags; only set this as needed in a main branch.
+    # Set to true to publish an unpublished notebook, or false to unpublish a published notebook.
+    # Omitting the publish setting leaves the publish-state unchanged (recommended for all non-primary branches).
+    publish: <Boolean>
+
+    # An optional array of absolute paths to other files in the Git repository that should be read and included with the notebook.
+    otherFiles: <String Array>
+
+    # An optional array of absolute paths to test parameter files in the Git repository.
+    # For example...
+    # testParameterFiles:
+    #   - /null-model/null-model.json
+    #   - /null-model/null-model-binary.json
+    testParameterFiles: <String Array>
```

Comment:
Currently, we don't do anything with the test parameter files, and it's not totally clear what format they should be.  Maybe they'll end up being related to https://papermill.readthedocs.io/en/latest/, but hasn't been decided yet.

Does that mean that for now, we leave this field out of the docs?

Reply:
Makes sense, I'll remove it

Reply:
@svonworl belated thought, if we end up not doing anything with test parameter files for notebooks, then we should remove the OpenData facet from search page.